deployment/manager/app/manager_config.json

```diff
@@ -6,13 +6,13 @@
     },
     "smartcity": {
       "styles": ":host > * {--or-app-color2: #F6F7FA; --or-app-color3: #1B1464; --or-app-color4: #662D91;}",
-      "logo": "/images/logo_smartcity.svg",
-      "logoMobile": "/images/logo-mobile_smartcity.svg"
+      "logo": "api/master/configuration/manager/image/logo_smartcity.svg",
```

Comment:
These paths should probably be left as before as it is clearer that they relate to the images dir in the deployment also

Reply:
After thinking about it, since I had to edit most of the frontend references to images, I am actually thinking of changing the frontend so that we don't actually require a change in the manager_config. So, the manager_config references would remain the same.

Reply:
It's fixed, but I don't include the `images` part, so it's just a reference to what the file should be within the `storageDir/manager/images` directory.

Reply:
Backtracking here but I've put all images in this demo `manager_config.json` to the API image endpoint for consistency as they should correctly resolve.